z01-default_modules.csh

```diff
@@ -1,21 +1,21 @@
-if ( ! $?USER_IS_ROOT ) then
+if ( $?USER_IS_ROOT ) then
```

Comment:
OK, this comment applies to all *mod*sh scripts, but I need to mention it now to avoid duplication of work:
* if we are to have a compatible way to transition from Lmod/6.6.x-cm70 towards Lmod/7.x-cm3, perhaps listening to the $LMOD_DISABLE variable would be useful, as per:
https://github.com/Bright-Computing/bic/issues/38

Essentially, this is no more work other than duplicating the `$USER_IS_ROOT` conditionals, with an extra exit clause for `$LMOD_DISABLE `. Do you think we go that way or, we drop #38 altogether?!